R/lineage.R

```diff
@@ -323,7 +323,7 @@ ipg2lin <- function(accessions, ipg_file,
     ipg_dt <- fread(ipg_file, sep = "\t", fill = T)
 
     accessions <- unique(accessions)
-    ipg_dt <- ipg_dt[Protein %in% accessions]
+    ipg_dt <- ipg_dt[.data$Protein %in% accessions]
```

Comment:
ah, data.table -- this package requires a different solution to the "no visible binding for global variable" warning. When subsetting, as in this line, we should define "Protein" in a separate .R file, called globals.R. The contents of this file will resemble the following:

```
#' @importFrom utils globalVariables
utils::globalVariables(c("species"))

```